tests/test_fr_page.py

```diff
@@ -0,0 +1,73 @@
+# Tests for parsing a page
+#
+# Copyright (c) 2021 Tatu Ylonen.  See file LICENSE and https://ylonen.org
+
+import json
+import unittest
+
+from unittest.mock import patch
+
+from wikitextprocessor import Wtp, Page
+from wiktextract.config import WiktionaryConfig
+from wiktextract.page import parse_page
+from wiktextract.thesaurus import close_thesaurus_db
+from wiktextract.wxr_context import WiktextractContext
+
+
+class FrPageTests(unittest.TestCase):
+    def setUp(self):
+        self.maxDiff = 20000
+        conf1 = WiktionaryConfig(
+            capture_language_codes=None,
+            capture_translations=True,
+            capture_pronunciation=True,
+            capture_linkages=True,
+            capture_compounds=True,
+            capture_redirects=True,
+            capture_examples=True,
+        )
+        self.wxr = WiktextractContext(Wtp(lang_code="fr"), conf1)
+        self.wxr.wtp.analyze_templates()
+        self.wxr.wtp.start_page("exemple")
+
+    def tearDown(self) -> None:
+        self.wxr.wtp.close_db_conn()
+        close_thesaurus_db(
+            self.wxr.thesaurus_db_path, self.wxr.thesaurus_db_conn
+        )
+
+    def runpage(self, text):
+        assert isinstance(text, str)
+        return parse_page(self.wxr, self.wxr.wtp.title, text)
+
+    def test_page1(self):
```

Comment:
I guess you copied the code from `tests/test_page.py` file, that file is not a good example... I have updated it: https://github.com/tatuylonen/wiktextract/commit/098461c5b1b60163fc19b603bbf67b2fc0de9eb6

And please use meaning name for test functions.

Reply:
Thanks for the review.

Yes, I didn't yet consider the final test form. I just wanted something quickly to run the parse_page function. Will improve!